server/game/cards/01_SOR/IAmYourFather.ts

```diff
@@ -0,0 +1,38 @@
+import AbilityHelper from '../../AbilityHelper';
+import { EventCard } from '../../core/card/EventCard';
+import { RelativePlayer, TargetMode, WildcardCardType } from '../../core/Constants';
+
+export default class IAmYourFather extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '0523973552',
+            internalName: 'i-am-your-father',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Deal 7 damage to an enemy unit unless its controller says "no"',
+            targetResolvers: {
+                targetUnit: {
+                    controller: RelativePlayer.Opponent,
+                    cardTypeFilter: WildcardCardType.Unit
+                },
+                opponentsChoice: {
+                    mode: TargetMode.Select,
+                    dependsOn: 'targetUnit',
+                    choosingPlayer: RelativePlayer.Opponent,
+                    choices: (context) => ({
+                        [`Let ${context.targets.targetUnit.title} take 7 damage`]: AbilityHelper.immediateEffects.damage({
```

Comment:
It's not a huge deal but I don't love the "Let" language on these, I would prefer it being framed as a choice of which effect to resolve:
- "<unit> takes 7 damage"
- "Opponent draws 3 cards"

That said if you like it as-is I won't block on it, just a suggestion